index.js

```diff
@@ -20,8 +20,9 @@
 
 'use strict';
 
-/* eslint object-shorthand: "off" */
+const Mosaic = require('@openstfoundation/mosaic.js');
 
+/* eslint object-shorthand: "off" */
```

Comment:
why? 🤔

Reply:
I don't know, it was already there.  I think it's no more valid. I have removed it.